config/fcos/v1_6_exp/validate_test.go

```diff
@@ -234,3 +234,117 @@ func TestValidateGrubUser(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateMountPoints(t *testing.T) {
+	tests := []struct {
+		in      Config
+		out     error
+		errPath path.ContextPath
+	}{
```

Comment:
Per https://github.com/coreos/butane/pull/449#discussion_r1191432117, let's also check `/var`, `/varnish` and `/foo/var`.

Reply:
No problem, have them added now